verilog/analysis/checkers/constraint_name_style_rule.cc

```diff
@@ -44,50 +42,64 @@ using verible::matcher::Matcher;
 // Register ConstraintNameStyleRule.
 VERILOG_REGISTER_LINT_RULE(ConstraintNameStyleRule);
 
-static constexpr absl::string_view kMessage =
-    "Constraint names must by styled with lower_snake_case and end with _c.";
-
-const LintRuleDescriptor& ConstraintNameStyleRule::GetDescriptor() {
+const LintRuleDescriptor &ConstraintNameStyleRule::GetDescriptor() {
   static const LintRuleDescriptor d{
       .name = "constraint-name-style",
       .topic = "constraints",
       .desc =
-          "Check that constraint names follow the lower_snake_case "
-          "convention and end with _c.",
+          "Check that constraint names follow the required name style "
+          "specified by a regular expression.",
+      .param = {{"pattern", kSuffix}},
   };
   return d;
 }
 
-static const Matcher& ConstraintMatcher() {
+absl::Status ConstraintNameStyleRule::Configure(
+    absl::string_view configuration) {
+  std::string pattern = kSuffix;
```

Comment:
I think this is redundant but the rationale is the following:

If at some point we add other parameters to the rule, but `pattern` is not provided, the `ParseNameValues` wouldn't set any value to `pattern`, so we would then need to set a default value. 

Avoid future problems by doing it already?